bundles/persistence/org.openhab.persistence.rrd4j/README.md

```diff
@@ -1,123 +1,183 @@
 # rrd4j Persistence
 
-The [rrd4j](https://github.com/rrd4j/rrd4j) Persistence service is based on a round-robin database.
+The [rrd4j](https://github.com/rrd4j/rrd4j) persistence service is based on a round-robin database.
 
-In contrast to a "normal" database such as db4o, a round-robin database does not grow in size - it has a fixed allocated size, which is used. 
+In contrast to a "normal" database such as db4o, a round-robin database does not grow in size - it has a fixed allocated size.
 This is accomplished by saving a fixed amount of datapoints and by doing data compression, which means that the older the data is, the less values are available. 
-The data is kept in several "archives", each holding the data for its set timeframe in the set granularity.
-The start point for all archives is the actually saved datapoint.
-So while you might have a value every minute for the last 8 hours, you might only have one every day for the last year.
+The data is kept in several "archives", each holding the data for its set timeframe at a defined level of granularity.
+The starting point for all archives is the actually saved data sample (Item value).
+So while you might store a sample value every minute for the last 8 hours, you might store the average per day for the last year.
 
-This service cannot be directly queried, because of the data compression. You could not provide precise answers for all questions. 
+This service cannot be directly queried, because of its data compression, which means that it cannot provide precise answers to all queries.
 
 NOTE: rrd4j is for storing numerical data only.
-Attempting to use rrd4j to store complex datatypes (e.g. for restore-on-startup) will not work.
+It cannot store complex data types.
 
-<!-- MarkdownTOC -->
+## Persistence Process
 
-- [Configuration](#configuration)
-    - [Datasource types](#datasource-types)
-    - [Heartbeat, MIN, MAX](#heartbeat-min-max)
-    - [Step\(s\)](#steps)
-    - [Example](#example)
-- [Troubleshooting](#troubleshooting)
+Round-robin databases (RRDs) have fixed length so called "archives" for storing values.
+Think of an archive as a "drawer" with a fixed number of "storage boxes" in it.
 
-<!-- /MarkdownTOC -->
+The persistence service reads data "samples" from the OpenHAB core at regular intervals, and these are then put into the storage boxes.
+Either a) the samples are stored singly directly into a box, or b) multiple samples are consolidated (using a consolidation function) into a box.
+
+The service starts by storing samples in the leftmost box in the drawer.
+Once the leftmost box is full, the service starts filling the next box to the right; and so on.
+Once the rightmost box in the drawer is full, the leftmost box is emptied, the content of all boxes is moved one box to the left, and new content is added to the rightmost box.
+
+So for example..
```

Comment:
I advocate for expanding this line to indicate that the example contains values that can vary, and then remove the "(say)" parentheticals from the lines below.

Reply:
Good idea, thank you.